src/Map/MapWrapper.web.js

```diff
@@ -17,12 +18,32 @@ const MapWrapper = ({
   filteredMarkers = [],
   viewCenter,
 }) => {
+  const markers = useMemo(() => {
+    if (!filteredMarkers) {
+      return []
+    }
+
+    return filteredMarkers.map(marker => (
+      <View lat={marker.lat} lng={marker.lng} onClick={marker.onPress} key={marker.key}>
+        <Image
+          resizeMode="contain"
+          source={marker.image}
+          style={[styles.markerImage, additionalStyles.markerImage]}
+        />
+      </View>
+    ))
```

Comment:
TIL: https://stackoverflow.com/questions/60453845/is-it-safe-to-usememo-for-jsx

Reply:
Our Runner would be much slower without this!

Reply:
Yeah, but I learned from that thread that `React.memo` won't stop re-renders on children. Wrapping it in a `useMemo`, you can be more granular on how to avoid re-renders of children, even if their props change.